src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+
+  if (days >= 7) {
+    return (dayRent * days - 50);
+  }
+
+  if (days >= 3) {
+    return (dayRent * days - 20);
```

Comment:
add variables instead of numbers and use them 
`const basicDiscount = 20;
  const extendedDiscount = 50;`